lstmcpipe/config_MC_prod.yml

```diff
@@ -27,6 +27,11 @@ obs_date: 20200629_prod5_trans_80
   # pro3 : `20190415`
 zenith: zenith_20deg  # Taken into account only when `prod_type: prod5` [ thus `obs_date` : 20200629_prod5(_trans_80) ]
 pointing: south_pointing
+
+# If the image modifier settings should be calculated on the fly
```

Comment:
The image modifier settings will only work when running a `dl1ab` stage, so I would put `dl1_reference_id`, `dl1_noise_tune_data_run`, `dl1_noise_tune_mc_run` close in the config file.